.travis.yml

```diff
@@ -17,5 +17,7 @@ gemfile:
 matrix:
   allow_failures:
     - rvm: ruby-head
-    - gemfile: gemfiles/rails_5.2.gemfile
```

Comment:
We still use the gemfiles and I think technically generate those from Appraisals, but we rely on github actions for CI now, so shouldn't need to change anything here. If you pull master down for this project, this file should now be gone.

Thanks for the change btw!